eject can not resolve babel plugins

**Describe the bug**
looks like `eject` has a problem with babel plugins resolution.

**To Reproduce**
Steps to reproduce the behavior:
1. `yarn eject`
2. See error

**Additional context**
```
$ yarn eject
yarn run v1.12.3
$ eject-ts @deity/falcon-ecommerce-uikit src/uikit
Ejecting @deity/falcon-ecommerce-uikit package into C:\A\Deity\my-app-8\client\src\uikit ...
(node:18464) UnhandledPromiseRejectionWarning: Error: Cannot find module '@babel/plugin-transform-typescript' from 'C:\A\Deity\my-app-8\client'
    at Function.module.exports [as sync] (C:\A\Deity\my-app-8\client\node_modules\resolve\lib\sync.js:58:15)
    at resolveStandardizedName (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\files\plugins.js:101:31)
    at resolvePlugin (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\files\plugins.js:54:10)
    at loadPlugin (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\files\plugins.js:62:20)
    at createDescriptor (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\config-descriptors.js:154:9)
    at items.map (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\config-descriptors.js:109:29)
    at createPluginDescriptors (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\config-descriptors.js:105:10)
    at alias (C:\A\Deity\my-app-8\client\node_modules\eject-ts\node_modules\@babel\core\lib\config\config-descriptors.js:63:49)
(node:18464) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejec
tion id: 2)
(node:18464) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 1.09s.
```

--- a/packages/eject-ts/index.js
+++ b/packages/eject-ts/index.js
@@ -64,7 +64,7 @@ async function eject(packageToEject, targetDir) {
   // while preserving original formatting and changing extension to .js
   convertToJs(ejectedFilesMeta);
 
-  // retrieve all project  files that can have imports to package being ejected
+  // retrieve all project files that can have imports to package being ejected
   const appFilesMeta = klawSync(appSrc, { nodir: true, filter: filterOnlyAppJsFiles(targetDir) });
 
   // and then replace imports inside them

--- a/packages/eject-ts/index.js
+++ b/packages/eject-ts/index.js
@@ -36,7 +36,7 @@ async function eject(packageToEject, targetDir) {
   const targetDirFullPath = path.join(process.cwd(), targetDir);
   const appSrc = path.join(process.cwd(), 'src');
   // assumption is that resolvedPackageDir points to dist folder that is one level deep inside root package dir
-  // and based on that assumption we can retreive package json and src dir
+  // and based on that assumption we can retrieve package json and src dir
   const resolvedPackageDir = path.dirname(require.resolve(packageToEject));
   const resolvedPackagePackageJsonPath = path.join(resolvedPackageDir, '../package.json');
   const resolvedPackagePackageJsonDependencies = require(resolvedPackagePackageJsonPath).dependencies;
@@ -58,20 +58,20 @@ async function eject(packageToEject, targetDir) {
   // copy source typescript files to target dir
   fs.copySync(packageSrcFiles, targetDirFullPath, { filter: filterOnlyTSSourceFiles });
 
-  // retreive all copied TS files metadata
+  // retrieve all copied TS files metadata
   const ejectedFilesMeta = klawSync(targetDirFullPath, { nodir: true });
   // once files are copied then we transform each of them by removing TS types
   // while preserving original formatting and changing extension to .js
   convertToJs(ejectedFilesMeta);
 
-  // retreive all project  files that can have imports to package being ejected
+  // retrieve all project  files that can have imports to package being ejected
   const appFilesMeta = klawSync(appSrc, { nodir: true, filter: filterOnlyAppJsFiles(targetDir) });
 
   // and then replace imports inside them
   replaceImports(appFilesMeta, packageToEject, targetDir);
 
   // as a final step install dependencies of package being ejected to project where it's ejected
-  // this logic fails when using yarn workspaces - https://github.com/yarnpkg/yarn/issues/4812
+  // this logic fails when using yarn workspace - https://github.com/yarnpkg/yarn/issues/4812
   const dependenciesToInstall = Object.keys(resolvedPackagePackageJsonDependencies).map(
     packageName => `${packageName}@${resolvedPackagePackageJsonDependencies[packageName]}`
   );
@@ -88,7 +88,7 @@ async function eject(packageToEject, targetDir) {
       chalk.green(
         `
          It appears that you've run eject command while using yarn workspaces in your project,
-         hence intallation failed due to known yarn issue (https://github.com/yarnpkg/yarn/issues/4812).
+         hence installation failed due to known yarn issue (https://github.com/yarnpkg/yarn/issues/4812).
          Package has been ejected, but if you'd have troubles running the project after it anyway, try running yarn install in the root dir.`
       )
     );
@@ -107,7 +107,7 @@ function convertToJs(ejectedFilesMeta) {
       plugins: [[require.resolve('@babel/plugin-transform-typescript'), { isTSX: true }], pluginRecast]
     });
 
-    // run prettier on transformed JS as recast preserves formatting such as whitespaces, but formats bit diferently thant prettier
+    // run prettier on transformed JS as recast preserves formatting such as whitespaces, but formats bit differently thant prettier
     const transformedContents = prettier.format(result.code, prettierOptions);
 
     // write transformed code to file

--- a/packages/eject-ts/index.js
+++ b/packages/eject-ts/index.js
@@ -104,7 +104,7 @@ function convertToJs(ejectedFilesMeta) {
     const fileContents = fs.readFileSync(filePath, 'utf8');
     // babel transforms Typescript to JS using recast to preserve as much original formatting as possible
     const result = babel.transformSync(fileContents, {
-      plugins: [['@babel/plugin-transform-typescript', { isTSX: true }], pluginRecast]
+      plugins: [[require.resolve('@babel/plugin-transform-typescript'), { isTSX: true }], pluginRecast]
     });
 
     // run prettier on transformed JS as recast preserves formatting such as whitespaces, but formats bit diferently thant prettier
@@ -126,10 +126,13 @@ function replaceImports(appFilesMeta, originalImport, replacementImport) {
     // run babel transforms that preserve formatting while replacing imports
     const result = babel.transformSync(fileContents, {
       plugins: [
-        '@babel/plugin-syntax-jsx',
-        '@babel/plugin-syntax-dynamic-import',
-        '@babel/plugin-syntax-class-properties',
-        ['babel-plugin-transform-rename-import', { original: originalImport, replacement: replacementImport }],
+        require.resolve('@babel/plugin-syntax-jsx'),
+        require.resolve('@babel/plugin-syntax-dynamic-import'),
+        require.resolve('@babel/plugin-syntax-class-properties'),
+        [
+          require.resolve('babel-plugin-transform-rename-import'),
+          { original: originalImport, replacement: replacementImport }
+        ],
         pluginRecast
       ]
     });

--- a/packages/falcon-ecommerce-uikit/src/Forms/Field.tsx
+++ b/packages/falcon-ecommerce-uikit/src/Forms/Field.tsx
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Field as FormikField, FieldProps as FormikFieldProps, FieldConfig, getIn } from 'formik';
 import { I18n } from '@deity/falcon-i18n';
-import { FormContext } from './Form';
 import { Validator } from './validators';
+import { FormContext } from './Form';
 
 type ValidateSequentially = { (validators: Validator[], label: string): FieldConfig['validate'] };
 const validateSequentially: ValidateSequentially = (validators, label) => value => {

